_includes/navbar.html

```diff
@@ -23,7 +23,7 @@
         </div>
       </form>
       <div class="nav navbar-right">
-          <a href="https://search.usa.gov/sites" class="navbar-brand"><i
+          <a href="https://search.gov/system-banner.html" class="navbar-brand"><i
```

Comment:
@dawnpm , I was going to suggest making these all relative links:
```html
          <a href="https://search.gov/system-banner.html" class="navbar-brand"><i
              class="icon-user icon-white"></i>&nbsp;Login</a>
...
          <a href="http://search.usa.gov/signup" class="navbar-brand">Sign up</a>
...
          <a href="https://search.gov/status.html" class="navbar-brand">System Status</a>
```
However, I'm seeing plenty of SEO articles that recommend absolute links, so let's keep 'em that way. (But let's update `http://search.usa.gov/signup` to `https`.)

Reply:
good catch on the http:, thanks.